pkg/neotest/signer.go

```diff
@@ -63,6 +71,11 @@ func (s *signer) SignTx(magic netmode.Magic, tx *transaction.Transaction) error
 	return (*wallet.Account)(s).SignTx(magic, tx)
 }
 
+// Account implements SingleSigner interface.
+func (s *signer) Account() *wallet.Account {
```

Comment:
Do we really need this wrapper? I performed this cast manually in the previous tests.

Reply:
Although it seems nice.